components/pricing/ForCollectiveCard.js

```diff
@@ -176,9 +176,13 @@ const ForCollectiveCard = () => {
               <FeeDescription>
                 <FormattedMessage
                   id="pricing.forCollective.hostFees"
-                  defaultMessage="<Fee>Host fees</Fee> may apply depending on your <FiscalHostLink>Fiscal Host</FiscalHostLink>"
+                  defaultMessage="{hostFees} may apply depending on your <FiscalHostLink>Fiscal Host</FiscalHostLink>"
                   values={{
-                    hostFees: FeeDataNoWrap,
```

Comment:
Here, this is the real problem. This line should be:

```es6
Fee: FeeDataNoWrap,
```

Reply:
Yes that was a mistake for sure with the variable name, but I think even when you put `Fee` it still don't work. And in fact the `lang:update` fails with, 

```
Value for "Fee" must be of type function
Error: Value for "Fee" must be of type function
```
Am I missing something there? 🤔